app/lib/run-playbook.sh

```diff
@@ -11,7 +11,11 @@ main() {
 	export BASEDIR=${BASEDIR:-.}
 
 	# Ensure we have HOME defined, otherwise set it manually
-	test -z "$HOME" && export HOME; HOME="$(getent passwd "$(id -un)" | cut -d: -f6)"
+	if [[ "$(uname)" == 'Darwin' ]]; then
+ 	   test -z "$HOME" && export HOME; HOME="$(dscl . -read /Users/$(id -un) NFSHomeDirectory | awk -F': ' '{print $2}')"
```

Comment:
Since you're already relying on the path `/Users/$(id -un)` you don't need all of it.

It's quite possible that you can't have your home path in other directory under macOS anyway :man_shrugging: 

```suggestion
 	   test -z "$HOME" && export HOME; HOME="/Users/$(id -un)"
```

Reply:
Nice and concise, I'm a fan. Implemented.